packages/app-builder/src/components/Form/FormInput.tsx

```diff
@@ -28,15 +28,18 @@ export const FormInput = React.forwardRef<
   React.ElementRef<typeof Input>,
   FormInputProps
 >(function FormInput({ type, ...inputProps }, ref) {
-  const name = useFieldName();
+  const { name, description } = useFieldName();
   const [meta] = useField<string>(name);
 
   return (
     <Input
       ref={ref}
       borderColor={meta.valid ? 'grey-10' : 'red-100'}
       {...inputProps}
-      {...getInputProps(meta, { type })}
+      {...getInputProps(meta, {
+        type,
+        ariaDescribedBy: description ? meta.descriptionId : undefined,
+      })}
```

Comment:
I need to handle `desccription` at the field level so `getInputProps(...)` can adapt the `aria-described-by` html attribute dynamically